Documentation/docs-site/content/usage/executor.en.md

```diff
@@ -0,0 +1,6 @@
+---
+title: "Executor"
```

Comment:
We shouldn't really write a usage guide to the executor; it's a component of our implementation. We should write a usage guide to controlling function execution, latency, throughput, memory costs etc.